pkg/tso/allocator_manager.go

```diff
@@ -217,17 +214,6 @@ func (am *AllocatorManager) getGroupIDStr() string {
 	return strconv.FormatUint(uint64(am.kgID), 10)
 }
 
-// GetTimestampPath returns the timestamp path in etcd.
-func (am *AllocatorManager) GetTimestampPath() string {
```

Comment:
ditto

Reply:
`GetTimestampPath` is used to print log when `created allocator manager`. I think 

```go
log.Info("created allocator manager",
		zap.Uint32("keyspace-group-id", group.ID))
```

is enough, do not have to know the TimestampPath.

Reply:
The comment is important.

Reply:
Got it. BTW, I found we still have some comments mentioning `KeyspaceGroupLocalTSPath`, please also remove it.